Assets/UnZENity-Core/Scripts/Adapter/IInteractionAdapter.cs

```diff
@@ -9,6 +9,7 @@ public interface IInteractionAdapter
         float GetFrameRate();
         GameObject CreatePlayerController(Scene scene, Vector3 position = default, Quaternion rotation = default);
         void CreateVRDeviceSimulator();
+        void LockPlayerInPlaceCutscene();
```

Comment:
Why not reusing function below? In the end it seems like we call the same VrController.Lock() function. Feel free to reuse until you need special treatment.

Reply:
This is different treatment as in the LockPlayerInPlace method it locks both transform and rotation. We want to rotation to stay so player can look around freely

Reply:
But in VR, I can move my head around normally. Isn't it the same as if I could rotate via Thumbstick? This will affect rotation as well, correct?

Reply:
Ah! You're correct - this is for the Player rotation and not camera.